src/eva/plotting/emcpy/plot_tools/figure_driver.py

```diff
@@ -65,9 +65,21 @@ def figure_driver(config, data_collections, timing, logger):
         if batch_conf:
             # Get potential variables
             variables = batch_conf.get('variables', [])
+
             # Get list of channels
             channels_str_or_list = batch_conf.get('channels', [])
             channels = parse_channel_list(channels_str_or_list, logger)
+            step_vars = channels
+            step_var_name = 'channel'
+            title_fill = ' Ch. '
+
+            # Get list of levels
+            levels_str_or_list = batch_conf.get('levels', [])
+            levels = parse_channel_list(levels_str_or_list, logger)
+            if levels != []:
+                step_vars = levels
+                step_var_name = 'level'
+                title_fill = ' Lv. '
 
             # Set some fake values to ensure the loops are entered
             if variables == []:
```

Comment:
Same comment as above.

Reply:
Change made here and elsewhere in the (previously) unchanged code.